app/client/components/payment/update/dd/directDebitInputForm.tsx

```diff
@@ -195,11 +195,22 @@ export const DirectDebitInputForm = (props: DirectDebitUpdateFormProps) => {
           />
         </FieldWrapper>{" "}
       </div>
+
       <Checkbox
-        onChange={(newValue) => setSoleAccountHolderConfirmed(newValue)}
+        onChange={(e) => setSoleAccountHolderConfirmed(e.target.checked)}
         checked={soleAccountHolderConfirmed}
         label="I confirm that I am the account holder and I am solely able to authorise debit from the account"
         required
+        value="I confirm that I am the account holder and I am solely able to authorise debit from the account"
+        cssOverrides={css`
+          div {
+            padding: 10px 0;
+          }
+
+          :focus {
+            box-shadow: none !important;
```

Comment:
It looks like this is being done as the default focus state doesn't match the existing controls on the form. It's not ideal to remove the focus state entirely though. Could we make it match the others instead or removing it?

Reply:
have added the correct focus state to the form fields

Reply:
Nice work 👍 Much better for accessibility.